lib/engine/game/g_18_esp/step/buy_sell_par_shares.rb

```diff
@@ -34,6 +34,12 @@ def can_buy_any?(entity)
           def can_ipo_any?(entity)
             @game.player_debt(entity).zero? && super
           end
+
+          def get_par_prices(_entity, corp)
+            super.reject do |p|
+              p.price == 100 || p.price == 95 if @game.phase.tiles.last == :yellow && @game.north_corp?(corp)
```

Comment:
This is a case where a game phase status would give clearer code `… if @game.phase.status.include('…')`.